.github/workflows/publish-next-major.yml

```diff
@@ -0,0 +1,232 @@
+# Description: This workflow runs unit + e2e tests, then publishes UI packages
+#              to `@next-release` NPM tag.
+#
+# Triggered by: merge to `next-major-release`
+
+name: Test and Publish / next-release
+
+concurrency:
+  group: e2e-${{ github.sha }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: [next-major-release]
```

Comment:
It should be 
```suggestion
    branches: [next-major-release/main]
```

Reply:
Or alternatively, what do you think about `next-release/main` to be concise?

Reply:
I think `next-major-release/main` is more precise because we are doing a **major** version bump with breaking changes.